analyses/rna-seq-expression-summary-stats/run-rna-seq-expression-summary-stats.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# PediatricOpenTargets 2021
+# Yuanchao Zhang
+set -e
+set -o pipefail
+
+# This script should always run as if it were being called from
+# the directory it lives in.
+# copied from https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/scripts/run_in_ci.sh
+script_directory="$(perl -e 'use File::Basename;
+ use Cwd "abs_path";
+ print dirname(abs_path(@ARGV[0]));' -- "$0")"
+cd "$script_directory" || exit
+
+# If results directory exists, remove the existing results.
+# If existing .gz files exist, gzip command asks for confirmation.
+# Design-wise, it is not surprising to remove previously generated results
+# before a new run.
+if [[ -d results ]]; then
+    rm -r results
+fi
```

Comment:
```suggestion
if [[ -d results ]]; then
    rm -r results
fi
```

I don't think this is necessary, and could potentially be unwanted, for instance, if we add another step or another analysis in which we add another type of file, rather than replace all files. With new runs, replacement should be sufficient so I think you can remove this.

Reply:
Thank you for the suggestion. I will remove this part.

Indeed, it a user runs this script from a wrong working directory, it will delete the `results` directory of their current working directory, which might be catastrophic.

I will also remove the same part in the `snv-frequencies` module.